### PR TITLE
Add a travis ci badge to README

### PR DESCRIPTION
**Pull request by shreyans800755**
Fixes https://github.com/boostorg/hof/issues/49

### CODE REVIEW

**Comment by shreyans800755** [comment]
@pfultz2 Can you please review this ?
I can also add appveyor badge. But I don't have access to the image link.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+doc/
```
Comment (pfultz2) [inline .gitignore]: Doc directory should not be ignored by git.

Comment (shreyans800755) [reply .gitignore]: Its generated for local docs, no ?

Comment (pfultz2) [reply .gitignore]: No, only doc/html is I believe.

**Comment by pfultz2** [comment]
Also, it would be better to put the gitignore changes in a separate PR.

**Comment by shreyans800755** [comment]
> Also, it would be better to put the gitignore changes in a separate PR.

Removed gitignore. I'll create another issue and create a new PR with that.

If you can give me the appveyor image link, I can add it here. And we can have both the badges.

**Comment by shreyans800755** [comment]
@pfultz2 
Go to appveyor site -> login -> Settings (Image 1) -> Badges (Image 2) -> URL (Image 2)

Image 1:
drawing


Image 2:
drawing 1

**Comment by pfultz2** [comment]
This is the URL:

https://ci.appveyor.com/api/projects/status/bjj27h3v3bxbgpsp/branch/develop

**Comment by shreyans800755** [comment]
@pfultz2 
Done.
You can view it here - https://github.com/shreyans800755/hof/tree/travis

**Comment by pfultz2** [comment]
Awesome thanks.

**Pull request merged**